### PR TITLE
update modal copy/content, updating action required screen

### PR DESCRIPTION
**Pull request by L03TJ3**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

About # (link your issue here)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

# Checklist:
- [ ] PR title matches follow: (Feature|Bug|Chore) Task Name
- [ ] My code follows the style guidelines of this project
- [ ] I have followed all the instructions described in the initial task (check Definitions of Done)
- [ ] I have performed a self-review of my own code
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] I have added reference to a related issue in the repository
- [ ] I have added a detailed description of the changes proposed in the pull request. I am as descriptive as possible, assisting reviewers as much as possible.
- [ ] I have added screenshots related to my pull request (for frontend tasks)
- [ ] I have pasted a gif showing the feature.
- [ ] @mentions of the person or team responsible for reviewing proposed changes

### CODE REVIEW

File: packages/good-design/src/core/modals/BasicModal.tsx [hunk 1]
```diff
@@ -57,8 +56,8 @@ const BasicModal: FC<BasicModalProps> = ({
   return (
     /* height 100vh is required so modal always shows in the middle */
     <NBModal isOpen={modalVisible} onClose={onClose} {..._modal} minH="100vh" bgColor={bgOverlay}>
-      <Box bgColor={bgContent} borderRadius="lg" h={height} width={width}>
-        <NBModal.Content w={"100%"}>
+      <Box borderRadius="lg" h={height} width={width} bgColor="white">
```
Comment (johnsmith-gooddollar) [inline packages/good-design/src/core/modals/BasicModal.tsx:59]: add prop for background, set it to white in the theme, re-use everywhere in this component

**Review by johnsmith-gooddollar — CHANGES_REQUESTED** [review]
1. fix small suggestion 
2. update versions
3. merge

File: packages/good-design/src/core/modals/BasicModal.tsx (hunk 1 above)
Comment (L03TJ3) [reply packages/good-design/src/core/modals/BasicModal.tsx:59]: withTheme hook? or just add a bgColor with default white as prop?

Comment (L03TJ3) [reply packages/good-design/src/core/modals/BasicModal.tsx]: done

**Review by johnsmith-gooddollar — APPROVED** [review]

**Pull request merged**